Xming: Clipboard fixes

Free the memory allocated for ReturnData in FlushXEvents(). 

from-xming-patch15


Copyright (C) Colin Harrison 2005-2008
http://www.straightrunning.com/XmingNotes/
http://sourceforge.net/projects/xming/ 

---
 xserver/hw/xwin/winclipboardxevents.c |    2 ++
 1 file changed, 2 insertions(+)

Index: xorg-server-1.5.3/xserver/hw/xwin/winclipboardxevents.c
===================================================================
--- xorg-server-1.5.3.orig/xserver/hw/xwin/winclipboardxevents.c
+++ xorg-server-1.5.3/xserver/hw/xwin/winclipboardxevents.c
@@ -718,6 +718,8 @@ winClipboardFlushXEvents (HWND hwnd,
 	      hGlobal = GlobalAlloc (GMEM_MOVEABLE, iConvertDataLen);
 	    }
 
+	  free (pszReturnData);
+
 	  /* Check that global memory was allocated */
 	  if (!hGlobal)
 	    {
